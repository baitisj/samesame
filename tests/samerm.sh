NAME=samerm
. ${srcdir}/tests/include_action.sh
