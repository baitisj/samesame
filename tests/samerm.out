samerm -A:
./work/test1/file2a
./work/test1/file3a

samerm -Z:
./work/test1/file2a
./work/test1/file1a

samerm -At:
./work/test1/file3a
./work/test1/file2a

samerm -Zt:
./work/test1/file3a
./work/test1/file1a

samerm -L:
./work/test1/file1a
./work/test1/file3a
